.storybook/Setup.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Canvas, useFrame } from 'react-three-fiber'
+
+export function Setup({ children }) {
+    return (
+        <Canvas colorManagement shadowMap camera={{ position: [-5, 5, 5] }} pixelRatio={window.devicePixelRatio}>
+            {children}
+            <ambientLight intensity={0.8} />
+            <pointLight intensity={1} color={'ffffff'} position={[0, 6, 0]} />
+        </Canvas>
+    )
+}
```

Comment:
You should add newlines at the end of files, most editors have this built-in or as a plugin when you press save. This is the reason why:
https://thoughtbot.com/blog/no-newline-at-end-of-file

The above page also has solutions for common editors.

Reply:
I thought we had prettier taking care of that, I may have skipped hooks by mistake